cdap-docs/admin-manual/source/installation/cdap-site.rst

```diff
@@ -23,6 +23,8 @@ see the :ref:`configuration-security` section.
    * - Parameter name
      - Default Value
      - Description
+   * - ``app.adapter.dir``
```

Comment:
You need to insert a line with a Default Value of ``/opt/cdap/master/plugins`` on it (begins with a dash, indented like the others) otherwise the build breaks. 
You could put the default as "``/opt/cdap/master/plugins`` or ``plugins``" to cover both cases.